pkg/runner/runner.go

```diff
@@ -76,7 +76,7 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	flag.BoolVar(&cfg.InstallTimescaleDB, "install-timescaledb", true, "Install or update TimescaleDB extension.")
 	flag.BoolVar(&cfg.ReadOnly, "read-only", false, "Read-only mode for the connector. Operations related to writing or updating the database are disallowed. It is used when pointing the connector to a TimescaleDB read replica.")
 	flag.BoolVar(&cfg.AdminAPIEnabled, "web-enable-admin-api", false, "Allow operations via API that are for advanced users. Currently, these operations are limited to deletion of series.")
-	envy.Parse("TS_PROM")
+	envy.Parse("PROMSCALE")
```

Comment:
can we leave TS_PROM in as well for legacy compatibility? Mark as deprecated in code? i.e. have both?